os/net/mac/ble/ble-l2cap.c

```diff
@@ -452,6 +452,13 @@ input_l2cap_frame_flow_channel(l2cap_channel_t *channel, uint8_t *data, uint16_t
 
   if((channel->rx_buffer.sdu_length > 0) &&
      (channel->rx_buffer.sdu_length == channel->rx_buffer.current_index)) {
+    if(channel->rx_buffer.sdu_length > packetbuf_remaininglen()) {
+      LOG_WARN("l2cap_frame: illegal L2CAP frame sdu_length: %d\n",
+               channel->rx_buffer.sdu_length);
+      return;
+    }
+
+
```

Comment:
One extra line of whitespace compared to the rest of the formatting in the file?